util/util.go

```diff
@@ -65,3 +65,11 @@ func JSONToMap(data []byte) (map[string]any, error) {
 
 	return decoded, nil
 }
+
+func Last[T any](list []T) T {
+	if len(list) == 0 {
+		var zeroVal T
+		return zeroVal
```

Comment:
the problem with this approach is it would return `0` for example for an integer null array which is a valid value.

Reply:
Or we can do this,
```
func Last[T any](list []T) (T, bool) {
	if len(list) == 0 {
		var zeroVal T
		return zeroVal, false
	}
	return list[len(list)-1], true
}
```